pkgs/applications/science/logic/logisim/default.nix

```diff
@@ -12,16 +12,11 @@ stdenv.mkDerivation {
   
   phases = [ "installPhase" ];
   
+  buildInputs = [makeWrapper jre];
```

Comment:
`makeWrapper` should be in `nativeBuildInputs` and `jre` shouldn't be needed as a build input at all because it is directly referenced in the wrapper you're creating.